src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -987,7 +987,11 @@ static void window_editor_objective_options_main_paint(rct_window* w, rct_drawpi
     ft = Formatter::Common();
     ft.Add<rct_string_id>(STR_STRING);
     ft.Add<const char*>(gS6Info.details);
-    gfx_draw_string_left_wrapped(dpi, gCommonFormatArgs, x, y, width, STR_BLACK_STRING, COLOUR_BLACK);
+    ScreenCoordsXY screenCoords;
+    screenCoords.x = x;
+    screenCoords.y = y;
+    gfx_draw_string_left_wrapped(dpi, gCommonFormatArgs, screenCoords, width, STR_BLACK_STRING, COLOUR_BLACK);
+
```

Comment:
```suggestion
    gfx_draw_string_left_wrapped(dpi, gCommonFormatArgs, { x, y }, width, STR_BLACK_STRING, COLOUR_BLACK);
```

You can create the objects in place following my suggestion. This function will probably have a `screenCoords` at some point, but then it will need to be used for all calls and you probably don't want to tackle it now.